app/helpers/react_helper.rb

```diff
@@ -62,6 +63,21 @@ def news_label(search)
     }
   end
 
+  def spelling_text(search, search_options)
+    return if search.spelling_suggestion.blank?
+
+    spelling_suggestion_links(search, search_options) do |suggested_query, suggested_url, original_url|
+      {
+        suggested: t(:showing_results_for,
+                     corrected_query: link_to(suggested_query,
+                                              suggested_url)),
+        original: t(:search_instead_for,
+                    original_query: link_to(search.query,
```

Comment:
do we care about word_otherword vs word.otherword?  Ruby convention is underscore right?

Reply:
I'm not entirely following, but if you're asking why `search.query` that's necessary as query is an attribute of the search object and `search.query` is how we must access it.  Is that what you're curious about?

On the other hand, `suggested_query`, `suggested_url`, `original_url` are variables being returned via the preexisting `spelling_suggestion_links` method defined here: https://github.com/GSA/search-gov/blob/main/app/helpers/spelling_suggestions_helper.rb#L2. (As an aside, we probably wouldn't write a method like that these days that yields multiple values, but it's already there, it works, it's covered by multiple tests, and it's likely unwise to refactor it as part of this effort, so I've just left it be.)

Reply:
Yes that explains.  Thanks!